### DISCUSSION TITLE
go get command not executing in my dispatch server

### DISCUSSION THREAD

**Discussion by Dcubeismynick** [learndevopsonline/learndevopsonline#1523] [Q&A]
go get command not executing in my dispatch server. As a result, my payment process is not successful

See the attached picture for error
image

![image](https://github.com/learndevopsonline/learndevopsonline/assets/125594439/3190d240-4207-4735-8ebe-e9a5ce9710e9)

**Comment by sreedharm07** [comment]
@Dcubeismynick 
after noon i tried in your machine but am not able to find the error even after following the correct steps, error may be from the os level.
delete the server and create new one and try bro it will work

**Comment by Dcubeismynick** [comment]
Ok bro
.thanks , I will do just that
I will recreate the payment and dispatch servers
